code/datums/martial/plasma_fist.dm

```diff
@@ -66,22 +66,22 @@
 	add_to_streak("H",D)
 	if(check_streak(A,D))
 		return 1
-	basic_hit(A,D)
-	return 1
+	add_logs(A, D, "punched with Plasma Fist")
```

Comment:
That will double up on the logs, though, no? You log it both here and in the parent function, unless I'm wrong.

Reply:
its how all the other martial arts do it, im just following precedent